_includes/assets/sass/_components.scss

```diff
@@ -36,4 +36,160 @@ COMPONENTS
 	h1, h3 {
 		margin-bottom: govuk-px-to-rem(21px);
 	}
+}
+
+// Tiles Block
+.tiles-block {
+
+	.tiles-block__tile {
+		margin-bottom: govuk-px-to-rem(39px);
+
+		hr {
+			display: inline-block;
+			background-color: $grey-footer;
+			height: govuk-px-to-rem(4px);
+			width: govuk-px-to-rem(60px);
+			border: none;
+			transition: 0.2s ease-in-out;
+			margin-top: 0;
+			margin-bottom: govuk-px-to-rem(28px);
+		}
+
+		&:hover {
+
+			hr {
+				transition: 0.2s ease-in-out;
+				width: 100%;
+				background-color: $red;
+			}
+		}
+	}
+}
+
+@media only screen and (min-width : $tablet) {
+	.tiles-block {
+
+		.tiles-block__tile {
+			min-height: govuk-px-to-rem(320px);
+			max-width: govuk-px-to-rem(336px);
+			-ms-flex-preferred-size: 0;
+			flex-basis: 0;
```

Comment:
Can we use auto prefixer or something so we don't have to manage the prefixes manually for our supported browsers?

Reply:
Absolutely. Installed an auto prefixer that prefixes with an expected base of browser-specific css. It's also very customisable as it is built from mixins :)

Reply:
Absolutely - I've added an autoprefixer. In this case, **-ms-flex-preferred-size: 0**; is not being auto prefixed, so it's still being included in the un-prefixed sass.

Let me know if you have any questions :)

Reply:
https://www.npmjs.com/package/autoprefixer

Reply:
Cool, this works but it looks like sass-autoprefixer hasn't been updated in 2 years and you can't set your browser support. Ideally we would use postcss autoprefixer but I don't mind if we push that back until later, as long as it works in the browsers we are supporting.

Reply:
We are using postcss autpoprefixer?